cds/modules/deposit/static/templates/cds_deposit/angular-schema-form/uiselect.html

```diff
@@ -21,6 +21,7 @@
                 {{select_model.selected.name}}
             </ui-select-match>
       <ui-select-choices
+        minimum-input-length="3"
```

Comment:
this is for all the fields :P IMHO we should pass it as an options (i.e. readonly)